### DISCUSSION TITLE
Capstone-ML Concrete Prediction

### DISCUSSION THREAD

**Discussion by daryraa** [teamalgoritma/community#350] [Q&A] [answered]
saya coba menggunakan neural network dan mendapat error seperti ini 
![image](https://user-images.githubusercontent.com/78478403/192154505-05e1dc46-9db5-4fdb-a3b5-1fd7287000b7.png)
karena apa ya?
dan ini  juga 
![image](https://user-images.githubusercontent.com/78478403/192178672-9292ba60-e3eb-49e5-8b47-5f7c49e9c402.png)

**Comment by idoali** [comment, marked as answer]
Selamat pagi pak. Kesalahan kemungkinan ada pada ukuran data yang masih belum pas pak. Data kita harus memiliki jumlah kolom yang sama dengan jumlah input node pada model deep learning. Jadi mohon bisa dicek terlebih dulu pak ukuran dari `test_x_s`, `test_x` dan `test_y`. 

Semoga bisa membantu pak

**Reply by daryraa** [reply]
okei mas terima kasih
